packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/DeferredEventEmitter.java

```diff
@@ -176,14 +176,19 @@ public void onMediaFileUploadSucceeded(int mediaId, String mediaUrl, int mediaSe
     }
 
     @Override
-    public void onMediaFileUploadFailed(int mediaId) {
-        setMediaFileUploadDataInJS(MEDIA_UPLOAD_STATE_FAILED, mediaId, null, 0);
+    public void onMediaFileUploadFailed(int mediaId, float progress) {
+        setMediaFileUploadDataInJS(MEDIA_UPLOAD_STATE_FAILED, mediaId, null, progress);
+    }
+
+    @Override
+    public void onMediaFileUploadPaused(int mediaId, float progress) {
+        setMediaFileUploadDataInJS(MEDIA_UPLOAD_STATE_PAUSED, mediaId, null, progress);
```

Comment:
I believe `MEDIA_UPLOAD_STATE_PAUSED` needs to be defined as `11` atop this file. Currently it is undefined.